generate-release-notes.test.js

```diff
@@ -137,3 +137,20 @@ test('ensure that replacements aren’t applied in commit titles', async () => {
 
 	expect(output).toEqual(expect.stringContaining('{date}'));
 });
+
+test('generates changelog using reverse optios', async () => {
+	const output = await generateReleaseNotes({
+		range: 'v3.1.0..v3.2.0',
```

Comment:
Why not just use the existing range? Does it have problems with #36?
```suggestion
		range,
```

Reply:
No, I will edit